src/server/common/helpers/certificates.js

```diff
@@ -53,9 +62,11 @@ export const uploadCertificateDetails = async (request, newCertificate) => {
   }
 
   let newCertificateList = []
-  const regex = /GBR-\d{4}-(CM|PM|SM)/g
-  const matches = regex.exec(newCertificate.certNumber)
-  if (matches && matches.length > 0) {
+  const isCertificateNumberValid = checkCertificateNumber(
+    newCertificate.certNumber
+  )
+
+  if (isCertificateNumberValid ?? bypassRegex) {
```

Comment:
is there any reason why we need to bypassRegex as opposed to add `CC|PS|SD` to the document number check?